python/GangaLHCb/Lib/GaudiRunApp/GaudiRunAppUtils.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
+import time
+from Ganga.Core.exceptions import GangaException
+from Ganga.Runtime.GPIexport import exportToGPI
+from Ganga.Utility.logging import getLogger
+from Ganga.Core.exceptions import ApplicationConfigurationError
+from .PythonOptsCmakeParser import PythonOptsCmakeParser
+
+logger = getLogger()
+
+def readInputData(optsfiles, app):
```

Comment:
I'm wanting to avoid any additional changes to the latter as this is heavily co-opted from other LHCb code which I'm just adopting and using here. I want to avoid changing the way it's called in-case I replace 1 for the other at some stage (although this is looking unlikely)